### DISCUSSION TITLE
[JOBS]: Error severity level on the requeue operation

### DISCUSSION THREAD

**Discussion by iborysenko** [roadrunner-server/roadrunner#1470] [General]
### Plugin

JOBS

### I have an idea!

Hi @rustatian 

Is it possible to add functionality to `requeue task` silently, without writing to the stderr ? 
Or configure severity level for this part ?  

----- 
For example my task must execute HTTP call , but destination server may return 429, so I just need to requeue this task with delay silently, and throw exception after 5 attempts.

**Comment by rustatian** [comment]
Hey @iborysenko 👋🏻 

You're not writing to the STDERR; you're sending a command to the RR to requeue the task. Here is the sample on how to requeue the task after 3 attempts (use headers): https://github.com/roadrunner-server/rr-e2e-tests/blob/master/php_test_files/jobs/jobs_err.php

**Comment by rustatian** [comment]
If you need to configure the logger specifically for the `jobs` (or `amqp`) you may use a `channels`: https://github.com/roadrunner-server/roadrunner/blob/master/.rr.yaml#L146

**Comment by iborysenko** [comment]
@rustatian 
yes, but RR always log something like:
```txt
 ERROR   jobs            jobs protocol error     {"error": "failed", "delay": 5, "requeue": true}
```
according to https://github.com/roadrunner-server/jobs/blob/master/protocol/error.go#L10

is not it ?

**Reply by rustatian** [reply]
Yeah, it might be the wrong severity level was used.
